pkgs/applications/misc/auto-multiple-choice/default.nix

```diff
@@ -23,14 +23,16 @@
 , poppler
 , auto-multiple-choice
 }:
-stdenv.mkDerivation rec {
+lib.fix (self: stdenv.mkDerivation rec {
```

Comment:
I believe, `mkDerivation` already knows how to find a fixed point if you pass a function to it instead of attrset.

Reply:
And it should handle package overrides!

Reply:
Done! All in separate commits now, to have a good per-package history.